src/index.ts

```diff
@@ -16,7 +16,7 @@ export interface CheckOptions {
   port?: number;
   path: string;
   recurse?: boolean;
-  linksToSkip?: string[];
+  linksToSkip?: string[] | ((link: string) => boolean);
```

Comment:
I omitted the async because I was getting this error when I included it:

![Screen Shot 2019-11-20 at 9 58 36 AM](https://user-images.githubusercontent.com/2289/69264553-62d1fa80-0b7c-11ea-88cb-c96868ab1851.png)

Reply:
So that's weird - is this VS Code?  Alternatively, `(link: string) => Promise<boolean>` gets us to the same place.

Reply:
Yeah it's VS Code.

I used your `Promise<boolean>` suggestion here and updated the tests and docs in https://github.com/JustinBeckwith/linkinator/pull/120/commits/a146f2c316e1878ee776afd091269eecbcedfcca